### PR TITLE
KEYCLOAK-18471 Added ID to admin event object.

### PR DESCRIPTION
**Pull request by artur-baltabayev**

### CODE REVIEW

File: model/jpa/src/main/java/org/keycloak/events/jpa/JpaEventStoreProvider.java [hunk 1]
```diff
@@ -146,7 +145,7 @@ public void close() {
 
     private EventEntity convertEvent(Event event) {
         EventEntity eventEntity = new EventEntity();
-        eventEntity.setId(UUID.randomUUID().toString());
+        eventEntity.setId(event.getId());
```
Comment (hmlnarik) [inline model/jpa/src/main/java/org/keycloak/events/jpa/JpaEventStoreProvider.java:148]: What if `event.getId()` would be `null`? Could this be changed to
```suggestion
        eventEntity.setId(event.getId() == null ? UUID.randomUUID().toString() : event.getId());
```

Comment (hmlnarik) [reply model/jpa/src/main/java/org/keycloak/events/jpa/JpaEventStoreProvider.java]: This comment applies to other places as well, could you fix those please?

Comment (artur-baltabayev) [reply model/jpa/src/main/java/org/keycloak/events/jpa/JpaEventStoreProvider.java]: Fixed.

**Review by hmlnarik — APPROVED** [review]
Thanks for the PR

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Sorry, I missed that there are multiple commits. Could you please squash those before merging?

**Comment by artur-baltabayev** [comment]
@hmlnarik Done.

**Review by hmlnarik — APPROVED** [review]
Thank you!

**Pull request merged**